index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <title>Portofolio-Desktop-Version</title>
+  </head>
+  <body>
+      <header class="nav-bac backgroun-color">
+        <h1><a href="#" class="my-logo">MRKA</a></h1>
+        <nav class="nav-bar">
+          <ul class="menu">
+            <li><a href="#parent-cards">Portofolio</a></li>
+            <li><a href="#about-content">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
+      <main>
+        <article id="page-header">
+          <section class="page-header-container">
```

Comment:
- [ ] I think it will be better if you reduce the spacing between your toolbar and your headline section so that anybody visiting your website will see firsthand the introductory content which is a good user experience 👍🏽. 

| Figma Design | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/193325755-88949966-a091-45cb-a126-c542c1f71a52.png)  | ![image](https://user-images.githubusercontent.com/63473584/193325682-644866c6-5b39-4a9e-9519-af98c660ed9e.png)  |

Reply:
@vickymarz 
thanks for your good and valuable feedback.
I will try my best to fix it and submit it again for reviwe.